lib/install.sh

```diff
@@ -52,9 +66,15 @@ fi
 
 # Copy the specific platform binary.
 platform=$(get_platform)
-arch="x86_64"
+arch=$(get_arch)
 bin_name="phylum-${platform}-${arch}"
-check_copy "${bin_name}" "${HOME}/.phylum/phylum" 
+
+if [[ "$arch" == "aarch64" ]]; then
+    cat "${bin_name}" > "${HOME}/.phylum/phylum"
```

Comment:
Should this be moved into `check_copy` itself? Seems like this will only get used for `aarch64`. It's _probably_ safe to use it for all architectures/platforms?

Reply:
Held off on that for now, since some of the `check_copy` calls are to directories as the destination.